main.go

```diff
@@ -183,6 +183,25 @@ func main() {
 		failf("Failed to get absolute project path: %s", err)
 	}
 
+	//
+	// Read the Scheme
+	var s *xcscheme.Scheme
+	var sc string
+	var conf string
+	{
+		s, sc, err = readScheme(absProjectPath, cfg.Scheme)
+
+		if err != nil {
+			failf("Failed to read schema: %s", err)
+		}
+
+		if cfg.Configuration != "" {
+			conf = cfg.Configuration
+		} else {
+			conf = s.ArchiveAction.BuildConfiguration
```

Comment:
The Xcode UI uses the Configuration of the Run action when Building.

Reply:
Yes I experienced that the "LaunchAction"'s configuration is used by default if we don't feed anything to the xcodebuild. But for some reason we already have a fallback to the archive configuration at `findBuiltProject`. Also we don't have the launchaction included in the `xscheme.go` If we wish to default to that we need to update the `xcode-project`

Reply:
The current solution is acceptable, please document it in the step.yml Configuration input description.

Reply:
Updated the description